README.md

```diff
@@ -1 +1,46 @@
-# Project-MicroReddit
\ No newline at end of file
+# Project-MicroReddit
+
+This is the project to learn about the database arrangement on Rails.
+
+
+## Built With
+
+-   Ruby
+-   Rubocop
+-   Ubuntu
+-   Rails
+
+## Prerequisities
+
+To get this project up and running locally, you must already have ruby and rails installed on your computer.
+
+## Getting Started
+
+**Setup**
+
+- Clone this repository with ```https://github.com/prernapriyali/Project-MicroReddit.git``` using your terminal or command line.<br>
+- Change to the project directory by entering ```cd Project-MicroReddit``` in the terminal<br>
+- Next run ```bundle install``` to install the necessary dependencies<br>
+- Run ```rails db:migrate``` to setup your local database.<br>
+- Finally run ```rails console``` to start the application.<br>
```

Comment:
- [ ] We didn't complete the instructions on how to run the app. Let's continue from where you stopped so as to make it more descriptive.

After running rails console:

- run `user = User.create(name: "your name", email: "your email")` to create a user
- run `post = Post.create(title: "your title",  body: "your body", user_id: 1)` to create a post
- run `comments = Comment.create(body: "your body", user_id: 1, post_id: 1)` to create a comment

- [ ] Lastly, I coudn't run the app without running `yarn install`. So do well to include it to the `setup`.